plugin-oracle/pom.xml

```diff
@@ -44,4 +44,11 @@
         </dependency>
     </dependencies>
 
+    <repositories>
```

Comment:
What's that for?

Reply:
So that the project works without the specific settings.xml. As long as you have credentials somewhere.